src/Vira/Page/BranchPage.hs

```diff
@@ -38,13 +47,26 @@ viewHandler repoName branchName = do
   pure $ W.layout cfg crumbs $ do
     viewRepoBranch cfg.linkTo repo branch jobs
 
+updateHandler :: RepoName -> Git.BranchName -> Eff App.AppServantStack (Headers '[HXRefresh] Text)
+updateHandler repoName branchName = do
+  repo <- App.query (St.GetRepoByNameA repoName) >>= maybe (throwError err404) pure
+  let glob = Git.BranchNameGlob (Git.unBranchName branchName)
+  branches <- liftIO $ Git.remoteBranchesByGlob glob repo.cloneUrl
+  App.update $ St.SetRepoBranchesA repo.name branches
```

Comment:
Wouldn't this discard other branches? Indeed it does:

<img width="629" alt="image" src="https://github.com/user-attachments/assets/846dde60-e233-40f6-9199-6c7fc060fff6" />

Reply:
Perhaps you should do the same refresh action without this glob stuff (not sure what that is for).  It is going to be a temporary solution anyway, per https://github.com/juspay/vira/issues/47#issuecomment-3014376804